src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -89,6 +90,12 @@ private BigInteger CalculateBonus(StoreView snapshot, BigInteger value, uint sta
             return value * sum * NeoHolderRewardRatio / 100 / TotalAmount;
         }
 
+        private static void CheckCandidate(StoreView snapshot, StorageKey key, CandidateState candidate)
+        {
+            if (!candidate.Registered && candidate.Votes.IsZero)
+                snapshot.Storages.Delete(key);
```

Comment:
Does we need to store a register one with 0 votes?

Reply:
Yes.